docs/src/main/asciidoc/http-reference.adoc

```diff
@@ -30,6 +32,36 @@ was chosen as it is the standard location for resources in `jar` files as define
 Quarkus can be used without Servlet, following this convention allows existing code that places its resources in this
 location to function correctly.
 
+[[from-mvnpm]]
+=== From mvnpm
+
+If you are using https://mvnpm.org/[mvnpm], as for the following JQuery dependency:
```

Comment:
is all this available from vert-http extension ? 

the doc actually dont state which extension you need to get these features (old bug but now that there is more added wanted to make sure it is clear)